colcon_ros_domain_id_coordinator/shell/ros_domain_id.py

```diff
@@ -105,6 +105,9 @@ async def generate_command_environment(  # noqa: D102
             else:
                 os.environ['ROS_DOMAIN_ID'] = self._default_id
         else:
+            os.environ['DISABLE_ROS_ISOLATION'] = '1'
+            os.environ['RMW_TEST_FIXTURE_DISABLE_ISOLATION'] = '1'
```

Comment:
Could we add some comments that describe which packages use which envars? (ament_cmake_ros vs rmw_test_fixture_implementation)